### ISSUE TITLE
Upgrade to Xenial produces ossec false posive for /usr/bin/mail

### ISSUE THREAD

**Issue by emkll** [freedomofpress/securedrop#4363]
## Description

Upgrade to Xenial produces a scary ossec syscheck alert:

```
OSSEC HIDS Notification.
2019 Apr 23 21:57:36

Received From: (app) 10.0.1.4->rootcheck
Rule: 510 fired (level 7) -> "Host-based anomaly detection event (rootcheck)."
Portion of the log(s):

Trojaned version of file '/usr/bin/mail' detected. Signature used: 'bash|file\.h|proc\.h|/dev/[^nu]' (Generic).

 --END OF NOTIFICATION
```
In this particular case, it seems due to the ruleset alerting on these changes https://askubuntu.com/a/605006

## Steps to Reproduce
Using prod vms:
* install 0.12.1 under trusty
* Upgrade the base OS to Xenial
* Observe the ossec alert above
## Expected Behavior
* ossec/syscheck alerts should not be triggered on system-provided binaries
## Actual behavior
* false-positive ossec/syscheck alert
## Comments

1. We should confirm if this is a one-time alert, or if it happens every `syscheck` run
2. We should see if updating ossec (and thus the ruleset) resolves the issue (there's an older WIP branch here: https://github.com/freedomofpress/securedrop/tree/ossec-3.2)

**Comment by eloquence** [comment]
I can confirm that I received this alert twice, once for app and once for mon, on 2/26, during an in-place upgrade of my SecureDrop 0.12.0 instance to Ubuntu 16.04.

**Comment by redshiftzero** [comment]
Filed ticket upstream: https://github.com/ossec/ossec-hids/issues/1720

**Comment by redshiftzero** [comment]
upstream has merged a PR fixing this: https://github.com/ossec/ossec-hids/pull/1765

not yet in a release, but 3.4.0 is in preparation, once released we can update to the new version of OSSEC and close this

**Comment by redshiftzero** [comment]
Fixed as of updating to OSSEC 3.6.0 in #5196
